clients/src/test/java/org/apache/kafka/clients/producer/internals/RecordAccumulatorTest.java

```diff
@@ -369,12 +375,11 @@ public void testAwaitFlushComplete() throws Exception {
         }
     }
 
-
     @Test
     public void testAbortIncompleteBatches() throws Exception {
         long lingerMs = Long.MAX_VALUE;
         final AtomicInteger numExceptionReceivedInCallback = new AtomicInteger(0);
-        final RecordAccumulator accum = new RecordAccumulator(4 * 1024 + DefaultRecordBatch.RECORD_BATCH_OVERHEAD, 64 * 1024,
+        final RecordAccumulator accum = new RecordAccumulator(128 + DefaultRecordBatch.RECORD_BATCH_OVERHEAD, 64 * 1024,
```

Comment:
I modified this because it seemed we were only testing that unsent batches were aborted. Now we verify that both sent and unsent batches are aborted with `abortIncompleteBatches`.